mpd/__init__.py

```diff
@@ -0,0 +1,215 @@
+import os
+from albertv0 import *
+from mpd import MPDClient
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "mpd: control your MPD"
```

Comment:
This is the pretty name. I guess "Music Player Daemon" would make sense. Put the brief description into the docstring.